search.php

```diff
@@ -6,49 +6,110 @@
  * @package shiro
  */
 
-get_header(); ?>
-
-<?php
+get_header();
 
 $wmf_results_copy = get_theme_mod( 'wmf_search_results_copy', /* translators: 1. search query */ __( 'Search results for %s', 'shiro-admin' ) );
 
 $template_args = array(
 	/* translators: Query that is currently being searched */
-	'h1_title' => sprintf( __( $wmf_results_copy, 'shiro' ), get_search_query() ),
+	'h1_title' => sprintf( __( $wmf_results_copy, 'shiro' ), get_search_query() ), // phpcs:ignore WordPress.WP.I18n.NonSingularStringLiteralText
 );
 
 get_template_part( 'template-parts/header/page-noimage', null, $template_args );
 
 ?>
 
+<?php if ( have_posts() ) : ?>
+	<div class="search-results__count mw-980">
+		<?php
+		$total_results = $wp_query->found_posts;
+		$paged = ( get_query_var( 'paged' ) ) ? get_query_var( 'paged' ) : 1;
+		$posts_per_page = get_query_var( 'posts_per_page' );
+		$first_result = ( $posts_per_page * $paged ) - $posts_per_page + 1;
+		$last_result = min( $total_results, $wp_query->post_count * $paged );
+		if ( $total_results === 1 ) {
+			printf( esc_html__( 'Showing 1 of 1 result', 'shiro' ) );
+		} else {
+			printf(
+				/* translators: 1. first result, 2. last result, 3. total results */
+				esc_html__( 'Showing %1$s - %2$s of %3$s results', 'shiro' ),
+				esc_html( $first_result ),
+				esc_html( $last_result ),
+				esc_html( $total_results )
+			);
+		}
+		?>
+	</div>
+
+	<div class="search-results__tabs mw-980">
+		<?php
+			$options = [
+				'all' => 'All',
```

Comment:
These labels would be nice to mark for translation, so they can be more useful on non-English sites. Could the labels be taken from the post type labels, since they're already internationalized there?

Reply:
Sure, added.